app/assets/stylesheets/tools.css.scss

```diff
@@ -83,4 +89,39 @@ ul.reproducibility_checklist {
     margin-right: 1em;
     display: inline;
   }
-}
\ No newline at end of file
+}
+
+// Pagination control
+.pagination {
+  margin-top: 1em;
+  margin-bottom: 2em;
+
+  a, span, em {
+    padding: 0.2em 0.5em;
+    display: block;
+    float: left;
+  }
+
+  .disabled {
+    display: none;
+  }
+
+  .current {
+    font-style: normal;
+    font-weight: bold;
+    background: #29abe2;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.